docs/JLBP-21.md

```diff
@@ -0,0 +1,32 @@
+# [JLBP-21] Separate tool classpath from product classpath
+
+The tools we use for Java projects are more often than not themselves
+written in Java. This includes build system such as Maven, javac, and Ant;
+runtime environments such as Tomcat, Hadoop, Beam, and App Engine; and editors
+such as jEdit, Eclipse, and IntelliJ IDEA. Tools like these should
+have their own classpaths from which they load their own code.
+
+It is critical not to mix the classpath of the tool with the classpath of the product.
+Dependencies of the tool should not be dependencies of the product.
```

Comment:
This line ("Dependencies of the tool...") seems redundant with the prior sentence.

Reply:
It is, I suppose; but sometimes it's helpful to say the same thing in different ways to get the point across.

Reply:
This contradicts all of your prior edit suggestions on the JLBPs to ensure conciseness...